run.sh

```diff
@@ -26,6 +26,8 @@ source ${projdir}/version.sh
 
 mkdir -p ${builddir}
 
+sudo apt install -y ${depends}
```

Comment:
This is not a nice thing to do inside a script. Nothing else here needs sudo or installs things on the users' host system.

The premise for this commit is also completely wrong. Build time dependencies *are* listed in the control files. It is the section called "Build-Depends", did you even look!? You can also list those dependencies using `dpkg-checkbuilddeps` and even auto-install them with `mk-build-deps`.

All of this info is a simple web search away, making this an embarrassingly bad patch. One that somehow 4 of y'all simply LGTM'd and now it gets to live forever in the git history even after you revert it (which you should do now). This could have been avoided if you had added me as a reviewer, or waited more than a single day between posting and merging..